packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -413,6 +413,40 @@ describe('<Autocomplete />', () => {
         expect(getAllByRole('button', { hidden: false })).to.have.lengthOf(5);
       }
     });
+
+    it('When the input box needs to expand downward, the listbox should remain open.', () => {
```

Comment:
This is also passing on the `next` branch. It should fail there. `handleClose` should be called once there.